src/components/progressiveImage/index.js

```diff
@@ -1,54 +0,0 @@
-import React from 'react';
```

Comment:
This file also not relevant to this PR. 
I think your environment is not synced or something.. 
These are files from a PR you submitted long time ago..